dotnet/targets/Xamarin.Shared.Sdk.targets

```diff
@@ -703,7 +705,7 @@
 			Condition="'$(_SdkIsSimulator)' != 'true' And '$(_PlatformName)' != 'macOS'"
 			DependsOnTargets="_ComputeVariables"
 			Inputs="@(_AssembliesToAOT)"
-			Outputs="@(_AssembliesToAOT -> '$(_AOTOutputDirectory)%(Arch)\%(Filename)%(Extension).o')">
+			Outputs="@(_AssembliesToAOT -> '%(ObjectFile)');@(_AssembliesToAOT -> '%(LLVMFile)');">
```

Comment:
@rolfbjarne are both ObjectFile and LLVMFile ITaskItem outputs of any task? I'm assuming those files will be produced by a task executed on macOS, so we need those to be an output prop of a task so they are copied to Windows to make the Inputs/Outputs check work, otherwise this target will always be executed from Windows.

Reply:
@emaf I changed this to add these files to `FileWrites`: https://github.com/xamarin/xamarin-macios/pull/12136/commits/d59c218a711ffcb89b1c4541ef473b3d5795ebbd, would that work?

Reply:
Yeah, I think that should work!